src/commands/collection/create.ts

```diff
@@ -8,30 +8,27 @@ export default class CollectionCreate extends Kommand {
 
   static flags = {
     help: flags.help(),
-    body: flags.string({
-      description: 'Collection mappings and settings in JS or JSON format. Will be read from STDIN if available',
-      default: '{}'
-    }),
     ...kuzzleFlags
   }
 
   static args = [
     { name: 'index', description: 'Index name', required: true },
-    { name: 'collection', description: 'Collection name', required: true }
+    { name: 'collection', description: 'Collection name', required: true },
+    { name: 'body', description: 'Collection mappings and settings in JS or JSON format. Will be read from STDIN if available' },
```

Comment:
```suggestion
    { name: 'body', description: 'Collection mappings and settings in JS or JSON format. Will be read from STDIN if available' }
```

Reply:
Actually this is intended, it's so much easier when you end with a trailing comma to add new element or copy paste any element